ui/src/core/xibo-forms.js

```diff
@@ -500,30 +500,216 @@ function permissionsMultiFormSubmit(id) {
     });
 }
 
+function createDisplayGroupMembersTable(dialog) {
+    var control = $(dialog).find(".controlDiv");
+    let context = {
+        'tableName' : '#displaysGroupsMembersTable',
```

Comment:
It would be good to get ESLint running over these - I think `'tableName'` should be without the single quotes as its a property name? `tableName` ?

Equally we should have `'` instead of `"` in `"displayGroupId"` for example.

Reply:
Enabling this on an older file...is interesting, but I will make needed changes in the code that's in this PR only, so changed file is still readable ;)